_protected/app/langs/it_IT/config/config.ini

```diff
@@ -0,0 +1,17 @@
+[language]
+lang = it
+dir = ltr
+charset = "utf-8"
+
+[language.information]
+name = Italiano
+Author = "Valentino Decaroli"
```

Comment:
Has to be lowercase only

Reply:
`[language.information]
name = Italiano
author = "Valentino Decaroli"`

Author <--- author
The same error is present in the Spanish translation:

_protected/app/langs/es_ES/config/config.ini

With all the sample files I had available, I used the only file that has the syntax error… 🥇 LOL³ :)
A new update is needed regarding the pH7-Internationalization language pack.

Thank you Pierre for your attention